examples/algorithms/writing_new/collect_indexes__complicated_real_example.cpp

```diff
@@ -81,26 +79,28 @@ void collect_indexes(R&& r, P p, std::vector<IdxType, Alloc>& res)
       r_with_idx,
       [&](eve::algo::iterator auto r_idx_it, eve::relative_conditional_expr auto ignore) mutable
       {
+        auto [elems_it, idxs_it] = r_idx_it;
```

Comment:
why the change ?

Reply:
Because we now need the idx iterator. `eve::compress_copy` is better than `eve::compress_store` since it has more opportunities to be optimized